src/TestRotations.java

```diff
@@ -0,0 +1,55 @@
+//: object/TestRotations.java
+
+import junit.framework.*;
+
+/**
+ * JUnit Class to test the rotation methods from the Quaternion and RotationMatrix classes.
+ *
```

Comment:
That wouldn't lead to duplication in the code? I would be testing that the rotation with a matrix gives the same result as the rotation with a quaternion in the `RotationMatrix` test, and that the quaternion rotation gives the same result as the matrix in the `Quaternion` test.